contracts/JBTerminalStore.sol

```diff
@@ -76,7 +76,7 @@ contract JBTerminalStore is ReentrancyGuard, IJBTerminalStore {
     /// @custom:param _projectId The ID of the project to get the balance of.
     /// @custom:param _token The token to which the balance applies.
     /// @custom:param _token The token to which the balance applies.
-    mapping(IJBPaymentTerminal => mapping(uint256 => mapping(address => uint256))) public override
```

Comment:
@xBA5ED cant we cast to IJBPaymentTerminal in this contract? does it need to be a Multi?

Reply:
(maybe we dont need to cast from `address` at all)